README.rst

```diff
@@ -383,3 +384,45 @@ them through the validator (without deliverability checks) like so:
 
     python3 email_validator/__init__.py --tests < test_pass.txt
 
+
+Usage for Gmail
+==============
+Sources:
+--------
+::
+        
+        https://support.google.com/a/answer/33386?hl=en
+        http://gmail-miscellany.blogspot.in/2012/08/wrong-email-gmail-dots-issue.html
+        http://gmail-tips.blogspot.in/2014/07/not-my-email.html
+        https://support.google.com/mail/answer/10313?hl=en&authuser=1
+
+Call:
+-----
+::
+
+        try:
+          response = validate_gmail(email)
+        except Exception, e:
+          # EmailNotValidError exception
+          pass
+
+Example:
+--------
+::
+
+        validate_gmail('P.r.esident.of.the@gmail.com')
+        {
+        'domain': 'gmail.com', 
+        'email_ascii': 'presidentofthe@gmail.com', 
+        'local': 'presidentofthe', 
+        'email': 'P.r.esident.of.the@gmail.com'
+        }
+
+Key names:
+-----------
+::
+
+        email - original email
+        email_ascii - normalised email
+        local - normalised local part
+        domain - normalised domain which is always gmail.com
```

Comment:
What about Google Mail for custom domains? I believe the same normalization rules apply there. It should be possible to tell when Google Mail is in use on a given domain based on the MX record (as mentioned in #3), in which case this feature could be applied there as well.